src/Image.php

```diff
@@ -911,32 +909,16 @@ public static function callBase(
         $instance,
         array $arguments
     ) {
-        if (self::$logger) {
-            self::$logger->debug('Called \'callBase\'', [
-                'name' => $name,
-                'instance' => $instance,
-                'arguments' => $arguments
-            ]);
-        }
+        Main::debug($name, array_merge(['instance' => $instance], $arguments));
```

Comment:
We could use: `Main::debug($name, ['instance' => $instance, 'arguments' => $arguments)]);` instead of the `array_merge`.